plugins/modules/redfish_storage_volume.py

```diff
@@ -293,6 +376,8 @@
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.six.moves.urllib.error import URLError, HTTPError
 from ansible.module_utils.urls import ConnectionError, SSLValidationError
+from ansible_collections.dellemc.openmanage.plugins.module_utils.utils import MANAGER_JOB_ID_URI, wait_for_redfish_reboot_job, \
```

Comment:
can we update the uri in documentation in the return sample section as well?

Reply:
Added